source/_integrations/ozw.markdown

```diff
@@ -99,3 +99,29 @@ LED colors on switches.
 | `node_id`              | yes      | Node id of the device to set configuration parameter to (integer).                                              |
 | `parameter`            | yes      | Parameter number to set (integer).                                                                              |
 | `value`                | yes      | Value to set for parameter. (String or integer value for list, string for bool parameters, integer for others). |
+
+## Events
+
+### ozw.scene_activated
+
+This event is fired upon scene activation. The data in the event will vary depending on your particular Z-Wave device, however, here is an example from a Zooz ZEN27 Dimmer when the down button is pressed 3 times.
```

Comment:
I think documenting events is great, however, this shows the result of an event. How would a user find out their own events when reading this?

Reply:
I figure documentation for seeing events belongs somewhere else as it would be universal to all of Home Assistant. However, each integration should document the events it fires so that users know to listen for them. With the "old Z-Wave", zwave.scene_activated was fired upon scene activation. It wasn't immediately obvious to me what to listen for to get scene events using ozwbeta. I asked on Discord and received this answer and felt it should be documented so that others with the same question don't have to use Discord to get this information.

You'll notice the Z-Wave Documentation has an entire page devoted to Events (https://www.home-assistant.io/docs/z-wave/events/). I assume OZW should have this information too.

Reply:
We are moving away from creating separate pages for everything. So a separate page, like the Z-Wave example, is definitely not the way to go.

Nevertheless, I'm confused by your answer... As it did not relate to my original question?

Reply:
> How would a user find out their own events when reading this? 

I use the Events panel on the dev tools page. I'm not sure if this is documented anywhere.